org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoJ11/mavenTychoJ11.parent/mavenTychoJ11.tests/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 1.0.0.qualifier
 Bundle-SymbolicName: mavenTychoJ11.tests; singleton:=true
 Bundle-ActivationPolicy: lazy
 Require-Bundle: mavenTychoJ11,
- org.junit;bundle-version="4.12.0",
+ org.junit.jupiter.api;bundle-version="[5.0.0,6.0.0)",
```

Comment:
The Oomph setup has a lower bound of 5.1. I think we had some reason with the JU extension. Just for correctness,  I'd like to see a 5.1 lower bound here. Practically it won't matter though.
(also locations below)

Reply:
done